streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -446,6 +446,13 @@ void print(final Serde<K> keySerde,
      * @param printed options for printing
      */
     void print(final Printed<K, V> printed);
+    
+    /**
+     * Merges the given streams into one larger stream. 
+     * @param a stream which is  to be merged.
+     * No guarantee of order
+     */
```

Comment:
Can you clean up the JavaDoc please? It's sound nitty, but this is was people see later on, so it's important.

Cf JavaDoc of https://github.com/apache/kafka/pull/3880/files#diff-86c4def0d4e763baa3d7df0d72ad99f1L499

Thx.